### PR TITLE
Implemented post question

### PR DESCRIPTION
**Pull request by mtbosio**
Implemented post comment page.

(For Nathan) In the NewComment.js file, I just set the name to be "Matt" because it wouldn't work without a name. Once you implement oauth you should be able to just replace that with the User's name

### CODE REVIEW

File: packages/react-frontend/src/NewComment.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      ...comment,
+      [event.target.id]: event.target.value,
+    }));
+  }
+
+  function handleSubmit(event) {
+    postComment(
+      {
+        body: comment.body,
+        author: "Matt", //just set to prefix name until it works with oauth
```
Comment (NathanKuhn) [inline packages/react-frontend/src/NewComment.js:21]: Will be able to do `req.session.name` after my changes.

File: packages/react-frontend/src/apis.js [hunk 2]
```diff
@@ -33,4 +33,19 @@ function postQuestion(question) {
   return promise;
 }
 
-export { fetchComments, fetchQuestions, getQuestion, postQuestion };
+function postComment(comment, questionID) {
+  const promise = fetch(`${url}/questions/${questionID}/comments`, {
```
Comment (NathanKuhn) [inline packages/react-frontend/src/apis.js:37]: Not for this CR, but after enabling frontend proxy for the backend, we should probably put a `/api/` in the backend endpoints.

File: packages/react-frontend/src/pages/NewQuestion.js [hunk 3]
```diff
@@ -37,7 +36,6 @@ function NewQuestion() {
 
   return (
     <>
-      <Navbar navbar>navbar</Navbar>
```
Comment (NathanKuhn) [inline packages/react-frontend/src/pages/NewQuestion.js:40]: Why was this removed?

Comment (mtbosio) [reply packages/react-frontend/src/pages/NewQuestion.js:40]: It was incorrectly in the NewQuestion.js file so I removed it.

File: packages/express-backend/models/comment-services.js [hunk 4]
```diff
@@ -50,11 +49,14 @@ function findCommentByAuthor(author) {
...
-    question.comments.push(commentToAdd._id);
-  });
+  const date = new Date();
+  const newComment = {
+    questionID: sanitizeHtml(questionID),
+    author: sanitizeHtml(comment.author),
+    body: sanitizeHtml(comment.body),
+    date: date.toDateString(),
+  };
+  const commentToAdd = new commentModel(newComment);
```
Comment (charlesmoreno32) [inline packages/express-backend/models/comment-services.js:59]: Comments work as intended. Thank you for fixing up any problems from my initial schema and endpoibts.

File: packages/react-frontend/src/NewComment.js (hunk 1 above)
Comment (mtbosio) [reply packages/react-frontend/src/NewComment.js:21]: Sick

File: packages/express-backend/models/comment-services.js (hunk 4 above)
Comment (mtbosio) [reply packages/express-backend/models/comment-services.js:59]: Gotchu

File: packages/react-frontend/src/apis.js (hunk 2 above)
Comment (mtbosio) [reply packages/react-frontend/src/apis.js:37]: Sounds good
